ScnSoft-Education/LeetCodeTraining.Tests/TwoSumSolutionTest.cs

```diff
@@ -0,0 +1,27 @@
+using LeetCodeTraining.TwoSum1;
+
+namespace LeetCodeTraining.Tests
+{
+    public class TwoSumSolutionTest
+    {
+        [Fact]
+      
+            public void TwoSumArrayWith2Nums()
```

Comment:
Please do aligning, public should be on the same left align as [Fact] attribute.